### ISSUE TITLE
'latest' build definitions

### ISSUE THREAD

**Issue by jasonkarns** [nodenv/node-build#145]
Should we have some 'latest' build definitions that are always pointing to latest releases? The nodejs distributions list includes:

![image](https://cloud.githubusercontent.com/assets/119972/14326239/2884ba28-fbfb-11e5-85f1-ba6c244580b3.png)

We could just have some build definitions that point to those locations, though that would mean the local install name of those nodes would be `node-latest` or `node-latest-v5.x`; since the version name would be taken from the build definition and not "updated" after installation to reflect the actual version number.

Alternatively, we could try some kind of after-install hook to rename the version once-it's installed.

Is this a good or bad idea? It seems the rbenv team have avoided this for a long time. Presumably they have reason? `nvm` has similar aliases like `node` which points to latest release and `5.0` which is latest 5.x release.

**Comment by searls** [comment]
👌 this would be great for automating dev machine setup. I have to imagine 90% of installs are "whatever latest LTS is"

**Comment by ELLIOTTCABLE** [comment]
Yeah, came here to effectively post the same thing — to be clear, it's a *terrible* idea to have it installed *as “latest”*, so this definitely needs the same thing that #159 brought up, by the way.

Previously, on `nvm`, I could simply “nvm install latest” or “nvm install stable” or “nvm install nightly”, and it would do the obvious thing — and *better yet*, on a new-ish machine, I could `nvm install v8` and `nvm install v6` to install the latest of each of those releases.

tl;dr seems to me nodenv / node-build is sorely missing *install* aliases — arguments you pass to `nodenv install` that *don't* install precisely what you typed, but an intelligent interpretation thereof. (I wouldn't mind if this were locked behind a flag or something, for backwards compatibility — `nodenv install -a/--approx v8`, or something?)

**Comment by ELLIOTTCABLE** [comment]
Just following-up on this — reinstalled Nodenv today, spent five minutes trying to figure out why `nodenv install v8` or `nodenv install latest` wasn't working, and ended up finding my own response to this Issue. 🤣

Is this just blocked by the overall lack of implementation for aliasing? Or is this a wontfix / unwanted behaviour? `(=`

**Comment by jasonkarns** [comment]
It's definitely not wontfix. I wouldn't say it's blocked, but the implementation is not straightforward. I've attempted a handful of alternate approaches that each have some annoyances and downsides. I should have some time soon to wrap it up, though.

**Comment by shaharmor** [comment]
bump :)
